platform-integration/src/main/java/com/softicar/platform/integration/database/structure/DatabaseStructureJsonFromClasspathExtractor.java

```diff
@@ -17,6 +17,11 @@ public class DatabaseStructureJsonFromClasspathExtractor {
 
 	private final String tablePackagePrefix;
 
+//	public static void main(String[] args) {
```

Comment:
We could add a comment above this comment. Like "Use this main method to blablabla"